### ISSUE TITLE
`oelint.var.improperinherit` false positive on `inherit_defer ${VARIABLE}`

### ISSUE THREAD

**Issue by fbertux** [priv-kweihmann/oelint-adv#883]
Hi!

I noticed the following behavior while running oelint in a project. I'm using `populate_sdk_base.bbclass `as an example to illustrate the issue.

When running:
```
oelint-adv --release latest openembedded-core/meta/classes-recipe/populate_sdk_base.bbclass
```
on OE-Core master with oelint-adv 9.8.0, I get:
```
populate_sdk_base.bbclass:8:error:oelint.var.improperinherit:'${SDK_CLASSES}' is not a proper bbclass name [branch:true]
populate_sdk_base.bbclass:48:error:oelint.var.improperinherit:'rootfs_${IMAGE_PKGTYPE}' is not a proper bbclass name [branch:true]
```

Both lines use `inherit_defer`, [bbclass](https://git.openembedded.org/openembedded-core/tree/meta/classes-recipe/populate_sdk_base.bbclass):
```bitbake
inherit_defer ${SDK_CLASSES}           # line 8
inherit_defer rootfs_${IMAGE_PKGTYPE}  # line 48
```

Based on [oelint.var.inherit](https://github.com/priv-kweihmann/oelint-adv/blob/master/docs/wiki/oelint.var.inherit.md), `inherit_defer` should be used when variables are involved, so this usage looks correct to me.

Is this the expected behavior of `oelint.var.improperinherit`, or is it a false positive? Seems that `oelint.var.improperinherit` shouldn't look at `inherit_defer` and only check `inherit`.

Am I doing something wrong?

Thanks!
